src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -7,24 +7,34 @@
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
         DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
this should be class-level constant too (`private static final`, outside of the method, and according to constant naming [convention ](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names))

Reply:
not fixed